### ISSUE TITLE
Undefined subroutine &CGI::link called

### ISSUE THREAD

**Issue by eserte** [mrdvt92/perl-Geo-Leaflet#1]
Test suite of 0.01 fails:
```
# $VAR1 = bless( {
#                  'lon' => '-0.09',
#                  'lat' => '51.5',
#                  'popup' => 'marker'
#                }, 'Geo::Leaflet::marker' );
# $VAR1 = [
#           bless( {
#                    'lon' => '-0.09',
#                    'lat' => '51.5',
#                    'popup' => 'marker'
#                  }, 'Geo::Leaflet::marker' ),
#           bless( {
#                    'lon' => '-0.11',
#                    'radius' => 500,
#                    'popup' => 'circle',
#                    'properties' => {
#                                      'fillColor' => '#f03',
#                                      'color' => 'red',
#                                      'fillOpacity' => '0.5'
#                                    },
#                    'lat' => '51.508'
#                  }, 'Geo::Leaflet::circle' ),
#           bless( {
#                    'coordinates' => [
#                                       [
#                                         '51.509',
#                                         '-0.08'
#                                       ],
#                                       [
#                                         '51.503',
#                                         '-0.06'
#                                       ],
#                                       [
#                                         '51.51',
#                                         '-0.047'
#                                       ]
#                                     ],
#                    'properties' => {},
#                    'popup' => 'polygon'
#                  }, 'Geo::Leaflet::polygon' )
#         ];
Undefined subroutine &CGI::link called at /home/cpansand/.cpan/build/2024110902/Geo-Leaflet-0.01-0/blib/lib/Geo/Leaflet.pm line 278.
# Looks like your test exited with 255 just after 9.
t/002_html.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 9 subtests passed 
```

**Comment by mrdvt92** [comment]
Unfortunately, I could not reproduce this issue on my platform (RPM: `perl-CGI-3.63-4`).  I assume that your CGI.pm does not understand `use CGI qw{link};` but, I have no idea what versions of `CGI.pm` do or do not support that syntax.  I am working on porting from `CGI.pm` to `HTML::Tiny 1.08` based on the recommendation at `CGI::Alternatives`.

**Comment by mrdvt92** [comment]
Fixed in v0.02 commit 50bac07 by moving from CGI.pm to HTTP::Tiny.
